eAbsentee/form/templates/confirmationspanish.html

```diff
@@ -48,6 +48,7 @@ <h1> Su aplicación ha sido confirmada. </h1>
         <a class="btn btn-block btn-social btn-facebook" href="https://www.facebook.com/sharer/sharer.php?kid_directed_site=0&sdk=joey&u=https%3A%2F%2Fwww.eabsentee.org%2F&display=popup&ref=plugin&src=share_button">
           <span class="fa fa-facebook"></span> Comparta en Facebook
         </a>
+        
```

Comment:
remove extra line (eventually, we'll set up linting and formatting as a CI so we'll have consistent formatting)
```suggestion
```